pom.xml

```diff
@@ -109,52 +106,52 @@
 
 	<build>
 		<plugins>
-				<plugin>
-    <groupId>org.jacoco</groupId>
-    <artifactId>jacoco-maven-plugin</artifactId>
-    <version>0.8.2</version>
-    <executions>
-    
-    <execution>
-    <id>jacoco-check</id>
-    <goals>
-        <goal>check</goal>
-    </goals>
-    <configuration>
-        <rules>
-            <rule>
-                <element>PACKAGE</element>
-                <limits>
-                    <limit>
-                        <counter>LINE</counter>
-                        <value>COVEREDRATIO</value>
-                        <minimum>0.50</minimum>
-                    </limit>
-                </limits>
-            </rule>
-        </rules>
-    </configuration>
-</execution>
+			<plugin>
+			    <groupId>org.jacoco</groupId>
+			    <artifactId>jacoco-maven-plugin</artifactId>
+			    <version>0.8.2</version>
+			    <executions>
+
+			    <execution>
+			    <id>jacoco-check</id>
+			    <goals>
+				<goal>check</goal>
+			    </goals>
+			    <configuration>
+				<rules>
+				    <rule>
+					<element>PACKAGE</element>
+					<limits>
+					    <limit>
+						<counter>LINE</counter>
+						<value>COVEREDRATIO</value>
+						<minimum>0.50</minimum>
```

Comment:
I ran into a problem because of minimum covergare. 
![image](https://user-images.githubusercontent.com/73487119/100388788-36ec1f00-2ff1-11eb-8a73-2a7f55d2a377.png)

Reply:
Solved